### ISSUE TITLE
Unable to log in - "ALL login fields are required!".

### ISSUE THREAD

**Issue by roke-julian-lockhart** [PrivacyDragon/Bookwyrm_Android#12]
As https://github.com/PrivacyDragon/Bookwyrm_Android/issues/4#issuecomment-1905189127 aforestated:

<blockQuote>

I'm unable to login too. It states

> ALL login fields are required!

in barely visible text, at the bottom.

**Comment by PrivacyDragon** [comment]
Was this after having filled out all fields?

**Comment by roke-julian-lockhart** [comment]
> https://github.com/PrivacyDragon/Bookwyrm_Android/issues/12#issuecomment-1906735018

@PrivacyDragon, you would've thought I'd verified that, right? Turns out that I misinterpreted the placeholder text for `bookwyrm.social` as a prefilled domain. Took me 7 tries to realize that. Now I'm just getting the standard "Username or password incorrect" so I'll just subscribe to the other issue - https://github.com/PrivacyDragon/Bookwyrm_Android/issues/4#issue-1323332944.
